en/user_guide/keyword_management.rst

```diff
@@ -0,0 +1,12 @@
+Keyword management
+==================
+
+Open Orchestra allows to attach keyword to content or media. They describe the content and allow to manage it.
+In the keyword panel, you can create or destroy a keyword.
+
+Usage
+-----
+
+When you fill meta for content or media, already created keyword can be choice or you can create new on the keyword field.
```

Comment:
when you fill the meda data for a content or media
what is meant by "can be choice" ? Can be chosen ?
"or you create new on the keyword field" -> alternatively, you can create new ones using the ``keyword`` field